src/lib/GameViews/GameReview.svelte

```diff
@@ -0,0 +1,204 @@
+<script>
+	import { onMount } from 'svelte';
+	import SubmitButton from '$lib/shared/Button/SubmitButton.svelte';
+
+	onMount(async () => {
+		const firstFormEmojis = document.forms[0].elements['emoji'];
+		const confirmBox = document.querySelector('.confirm-container');
+		const yesBtn = document.querySelector('.yes-btn');
+		const secondForm = document.querySelector('.confirm-container');
+
+		firstFormEmojis.forEach((el) => {
+			el.addEventListener('click', () => {
+				confirmBox.style.display = 'block';
+				confirmBox.scrollIntoView({ behavior: 'smooth' });
+			});
+		});
+
+		yesBtn.addEventListener('click', () => {
```

Comment:
Kan dit niet met een `on:click`? of werkt dit dan niet met de `onMount`

Reply:
En voor het andere `click` event geld dan natuurlijk hetzelfde.

Reply:
Jaa, top. Heb heel stuk weg kunnen halen. Thanks.